lib/html/includes.js

```diff
@@ -640,6 +640,17 @@ var _MiniProfiler = (function() {
   };
 
   var bindDocumentEvents = function bindDocumentEvents() {
+    document.addEventListener("turbo:before-visit", e => {
+      window.MiniProfilerContainer = document.querySelector('body > .profiler-results')
+      if(!e.defaultPrevented) window.MiniProfiler.pageTransition()
+    })
+
+    document.addEventListener("turbo:load", e => {
+      if(window.MiniProfilerContainer) {
```

Comment:
could you also enhance `unbindDocumentEvents` to remove these events?
I think that will remove a memory leak.

Reply:
Done.